packages/client/src/components/Chat/index.tsx

```diff
@@ -1,24 +1,42 @@
 import React, { useEffect, useState } from "react";
-import { useSocket } from "../../contexts";
+
+import { useSocket, useTheme } from "../../contexts";
 import ChatBody from "./ChatBody";
 import ChatEntry from "./ChatEntry";
 
-export type Message = {text: string; id: string;};
+import styles from "./Chat.module.scss";
 
-export default function Chat() {
-    const { socket } = useSocket();
-    const [messages, setMessages] = useState<Message[]>([]);
+export type Message = { text: string; id?: string; colorOverride?: Color };
 
-    useEffect(() => {
-        socket.on("message broadcast", (message) => { console.log(message); console.log(messages); setMessages([...messages, message]) })
-    }, [messages]);
+export default function Chat() {
+  const { socket } = useSocket();
+  const { theme } = useTheme();
+  const [messages, setMessages] = useState<Message[]>([
+    {
+      id: "Server",
+      text: "Welcome to d'chat! Don't be nob!!!",
```

Comment:
Perhaps we should alter this language to be more affirmative?  Something like "Be pro!" rather than a warning